app/views/layouts/_footer.html.erb

```diff
@@ -1,10 +1,49 @@
-<footer>
-  <div id="footer-container" class="container <%= "centered-footer" unless current_page.include?("stories-show") %>">
-    <div class="inner-footer-container">
-      <a href="/">Home</a> <a href="/about">About</a> <a href="/privacy">Privacy Policy</a>
-      <a href="/terms">Terms of Use</a> <a href="/contact">Contact</a> <a href="/code-of-conduct">Code of Conduct</a><br />
-      <%= community_qualified_name %> copyright <%= copyright_notice %>&nbsp; 🔥
+<footer class="crayons-footer">
+  <div id="footer-container" class="crayons-layout crayons-footer__container">
+    <div class="crayons-footer__info">
+      <div><a href="/" class="crayons-footer__logo" aria-label="<%= community_name %> Home"><%= logo_svg %></a></div>
+      <p><%= SiteConfig.community_description %></p>
+      <div><%= render partial: "layouts/social_media" %></div>
+      <p class="fs-xs color-base-60"><%= community_qualified_name %> copyright <%= copyright_notice %></p>
     </div>
+
+    <nav class="crayons-footer__nav">
+      <div>
+        <a href="/" class="crayons-link crayons-link--block">Home</a>
+        <% if user_signed_in? %>
+          <a href="<%= readinglist_path %>" class="crayons-link crayons-link--block">Reading list</a>
+        <% end %>
+        <a href="<%= listings_path %>" class="crayons-link crayons-link--block">Listings</a>
+        <a href="<%= pod_path %>" class="crayons-link crayons-link--block">Podcasts</a>
+        <a href="<%= videos_path %>" class="crayons-link crayons-link--block">Videos</a>
+        <a href="<%= tags_path %>" class="crayons-link crayons-link--block">Tags</a>
+        <% unless user_signed_in? %>
+          <a href="/enter" class="crayons-link crayons-link--block fw-bold">
+            Sign In/Up
+          </a>
+        <% else %>
+          <a href="/new" class="crayons-link crayons-link--block fw-bold">Write a post</a>
+        <% end %>
+      </div>
+
+      <div>
+        <a href="/code-of-conduct" class="crayons-link crayons-link--block">Code of Conduct</a>
+        <a href="/faq" class="crayons-link crayons-link--block">FAQ</a>
+        <a href="/about" class="crayons-link crayons-link--block">About</a>
+        <a href="/privacy" class="crayons-link crayons-link--block">Privacy policy</a>
+        <a href="/terms" class="crayons-link crayons-link--block">Terms of use</a>
+        <a href="/contact" class="crayons-link crayons-link--block">Contact</a>
+      </div>
+
+      <div>
+        <a href="/sponsors" class="crayons-link crayons-link--block">Sponsors</a>
+        <% if SiteConfig.shop_url.present? %>
+          <a href="<%= SiteConfig.shop_url %>" class="crayons-link crayons-link--block"><%= community_name %> Shop</a>
+        <% end %>
+      </div>
+    </nav>
+
+    <img src="<%= asset_path("footer-sloan.png") %>" alt="" width="52" height="120" class="crayons-footer__mascot">
```

Comment:
this should probably be generalized somehow, as not all communities will have Sloan as a mascot

Reply:
We can do that in another PR as part of the Forem initiative for generalizing things.

Reply:
Agree but I have no idea how to "generalize" it... I mean it's hard to expect every community will have this kind of "hanging mascot" for footer... On the other hand I think it's a nice touch and wouldn't like to drop it from DEV for sake of generalization... Any thoughts? Ideas? Should we maybe have some sort of "community.scss" file which could potentially include all these kind of community-specific style overwrites? Not sure if I like it, but I also can't think of an alternative...

Reply:
cc @benhalpern thoughts on this?

Reply:
This wouldn't be a style override.

Would end up being like `<%= cloudinary(SiteConfig.footer_mascot_image_url) %>`... We already do that with other mascot instances via `mascot_image_url` so this is a specialized config.

It will require a separate PR. It's straightforward but definitely outside of scope for a designer so I'll whip it up....right now and will be ready when you come back to this tomorrow.

Reply:
Okay cool, #8692 solves this problem so we can get that merged asap... Note that I named it `mascot_footer_image_url` instead of what I wrote above.